### DISCUSSION TITLE
info of the base image

### DISCUSSION THREAD

**Discussion by Chunshan-Theta** [infiniflow/ragflow#1434] [Q&A] [answered]
I check the [dockerfile](https://github.com/infiniflow/ragflow/blob/main/Dockerfile), and there show the base image of ragflow is [infiniflow/ragflow-base:v2.0](https://hub.docker.com/layers/infiniflow/ragflow-base/v2.0/images/sha256-640c8e4ab9b1d7be5cae4fcee5fdf1600abc32e4b1387baa4c5f834d8b5c19e7?context=explore)
where can I get the detail about this image?

**Comment by KevinHuSh** [comment]
You can build a docker image from [scratch](https://github.com/infiniflow/ragflow/blob/8964817d726a9bc44065cf6a17a4d388fc583161/docs/references/faq.md#11-how-to-build-the-ragflow-image-from-scratch).

**Reply by Chunshan-Theta** [reply]
thank for reply.
But i want to know the detaile about `FROM infiniflow/ragflow-base:v2.0` in Dockerfile.
Any setting be applied in this image?

**Reply by KevinHuSh** [reply, marked as answer]
I'm sorry, None. ragflow-base:v2.0 was not build by Dockfile.

**Comment by Chunshan-Theta** [comment]
Update: I tried the this Dockerfile, built from Ubuntu:22.04, is work: [URL](https://github.com/infiniflow/ragflow/blob/main/Dockerfile.scratch)
